src/Arcus.Testing.Tests.Integration/XunitV2Using.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace Xunit.Abstractions
+{
+    // TODO: Remove this empty namespace once all tests have removed their using.
```

Comment:
This file is added to reduce the amount of changed files in the PR. Otherwise, there would be a lot more files that need to be altered by removing the `Xunit.Abstractions` namespace, as this is not available anymore in xUnit v3. By creating an empty namespace ourselves, we do it more gradually.